sql/moz-fx-data-shared-prod/firefoxdotcom_derived/gclid_conversions_v1/bigconfig.yml

```diff
@@ -8,7 +8,7 @@ tag_deployments:
   - column_selectors:
     - name: moz-fx-data-shared-prod.moz-fx-data-shared-prod.firefoxdotcom_derived.gclid_conversions_v1.*
     metrics:
-    - saved_metric_id: composite_key_uniqueness_2_column
+    - saved_metric_id: composite_key_uniqueness_2_column [fail]
```

Comment:
This would fail as there is no saved metric defined with that name.

Reply:
how do I make sure that the check fails?

Reply:
@kik-kik , if this uniqueness check fails, does the job fail or do we just get a warning?

Reply:
so this `composite_key_uniqueness_2_column` saved metric would not result in a failure, this would just cause an issue to be opened in Bigeye.